DESCRIPTION

```diff
@@ -1,24 +1,11 @@
 Package: jaspTTests
 Type: Package
 Title: T-Tests Module for JASP
-Version: 0.11.0
-Date: 2019-08-31
+Version: 0.13.0
+Date: 2020-10-26
 Author: JASP Team
 Website: jasp-stats.org
 Maintainer: JASP Team <info@jasp-stats.org>
 Description: This module offers the standard Frequentist and Bayesian T-Tests analyses.
 License: GPL (>= 2)
-Encoding: UTF-8
-Imports:
-  BayesFactor,
-  BSDA,
-  car,
-  ggplot2,
-  jaspBase,
-  jaspGraphs,
-  logspline,
-  plotrix,
-  plyr
-Remotes:
-  jasp-stats/jaspBase,
-  jasp-stats/jaspGraphs
+Imports: gghalves
```

Comment:
You should not be overwriting this. I get that this is probably because you rebuild the DESCRIPTION and NAMESPACE file automatically when installing the package to JASP, but I assume these parts were added manually, therefore automatically generating these files from `Description.qml` is no longer the course of action here. Am I right, @TimKDJ @vandenman?

I think you can instead just add the new package to imports in this file, and remove the `Package{}` request from the `Description.qml` file. But again, wait for instructions from @TimKDJ or @vandenman..

Reply:
If you're using a nightly to develop, don't check the Regenerate option:

![image](https://user-images.githubusercontent.com/21319932/97277510-dd010980-1838-11eb-8b38-999d3189f4c1.png)

that should leave the description and namespace unchanged. Then you can make your changes manually. Otherwise, you have to move all the changes to `Description.qml`, but I would advise against that.

Reply:
Yeah `Package {}` and the `Regenerate package metadata every time` checkbox still need to be removed; they shouldn't be used

Reply:
Ok, I will try to fix that!